op-batcher/batcher/config.go

```diff
@@ -74,6 +75,13 @@ func (c *CLIConfig) Check() error {
 	if c.RollupRpc == "" {
 		return errors.New("empty rollup RPC URL")
 	}
+	if strings.Contains(c.RollupRpc, ",") || strings.Contains(c.L2EthRpc, ",") {
+		rollupUrls := strings.Split(c.RollupRpc, ",")
+		ethUrls := strings.Split(c.L2EthRpc, ",")
+		if len(rollupUrls) != len(ethUrls) {
+			return errors.New("number of rollup and eth URLs must match")
+		}
+	}
```

Comment:
You can shorten this to
```suggestion
	if strings.Count(c.RollupRpc, ",") != strings.Count(c.L2EthRpc, ",") {
		return errors.New("number of rollup and eth URLs must match")
	}
```